h/migrations/env.py

```diff
@@ -4,12 +4,10 @@
 from alembic import context
 from sqlalchemy import engine_from_config, pool
 
+from h import models  # noqa
```

Comment:
This change has lost the comment about why `models` is being imported here. From the commits I can see this was intentional but I'm not clear why. Is it not still applicable?

Reply:
Fixed